.github/workflows/build.yml

```diff
@@ -0,0 +1,17 @@
+name: Build Cloud SDK Documentation
+
+on:
+  pull_request:
+    branches: [documentation]
+
+jobs:
+  build:
+   runs-on: ubuntu-latest
+   steps:
+     - uses: actions/checkout@v1
+     - uses: actions/setup-node@v1
+       with:
+         node-version: '12.x'
+         run: |
+          npm ci
+          npm run build
```

Comment:
Could you please share more insights here?
What does this script do here?
I would expect the `build` (or the old name `check`) tests something to ensure that merging the PR does not break anything + brings the value.

Reply:
This has the sole purpose of ensuring the PR can be built which in most cases means it is deployable after it's merged. It will catch inconsistencies in config files like `sidebar.js`, `docusaurus.config.js`, and some syntax errors for MD and JS files. Some runtime bugs will certainly slip through, but that is good enough test for now. If we will identify some sources of failure for docs we'll make sure to add more tests like checking links to work, linting Markdown, etc.